helm/templates/network-policy-app-ingress.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-app-ingress
```

Comment:
There's a fix to the names but I'm guessing that's coming in another PR

Reply:
let me try to include it, its hard, because sometimes the commits have multiple things